_rules/image-no-text-0va7u6.md

```diff
@@ -209,10 +209,30 @@ This `svg` element does not have `image` element descendants.
 </svg>
 ```
 
+#### Inapplicable Example 3
+
+This `canvas` element does not use [drawImage][] to render an image.
+
+```html
+<canvas id="canvas"> Your browser does not support the HTML5 canvas element. </canvas>
+
+<script>
+	const canvas = document.getElementById('canvas')
+	canvas.width = 400
+	canvas.height = 400
+	const ctx = canvas.getContext('2d')
+	ctx.font = 'bold 32pt Times'
+	ctx.fillStyle = '#c2c'
+	ctx.fillText('Welcome!', 100, 100)
+</script>
```

Comment:
Non sure how "customizable" this text is. Given the SC, if the text is not customizable by user, we shouldn't have an Inapplicable example with it.
Can still be just drawing a circle or whatever without text.

Reply:
Fair point. I could make the example customisable but I don't think we need a lot of scripting to convey the message that not all `canvas` elements are inapplicable